src/DependencyInjection/MonologExtension.php

```diff
@@ -292,6 +295,43 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 ]);
                 break;
 
+            case 'mongodb':
```

Comment:
DX improvement: throw an actionable exception when the mongodb lib is not installed.


```suggestion
            case 'mongodb':
                if (!class_exists('MongoDB\Client')) {
                    throw new \RuntimeException('The MongoDB Client library is not available. Try running "composer require mongodb/mongodb".');
                }
```